src/components/Loader/SliderLoader.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Container from '../shared/Container';
+import {SampleTitle, SampleSlider, SampleSlide} from './styles';
+
+export default function SliderLoader() {
```

Comment:
Same comment with the `FeaturedLoader` I think this should be under `Slider`? Since it is only used there?